docs/guides/extensions/panels-create-items.md

```diff
@@ -0,0 +1,566 @@
+
+Panels are used in dashboards as part of the Insights module. As well as read-only data panels, they can be interactive
+with form inputs. In this guide, you will create a panel that automatically generates a form based on a collection's
+fields, and allows item creation from an Insights dashboard.
+
+![A panel shows a form called Add Customer. It has a name, surname, and phone number text input.](https://marketing.directus.app/assets/5385ebc9-e3d9-4f75-9ec8-86bd54589d56.png)
+
+## Install Dependencies
+
+Open a console to your preferred working directory and initialize a new extension, which will create the boilerplate
+code for your operation.
+
+```
+npx create-directus-extension
+```
+
+A list of options will appear (choose panel), and type a name for your extension (for example,
+`directus-panel-internal-form`). For this guide, select JavaScript.
+
+Now the boilerplate has been created, open the directory in your code editor.
+
+## Specify Configuration
+
+Panels have two parts - the `index.js` configuration file, and the `panel.vue` view. The first part is defining what
+information you need to render the panel in the configuration.
+
+Open `index.js` and change the `id`, `name`, `icon`, and `description`.
+
+```js
+id: 'panel-internal-form',
+name: 'Internal Form',
+icon: 'view_day',
+description: 'Output a form to insert data into a collection.',
+```
+
+Make sure the `id` is unique between all extensions including ones created by 3rd parties - a good practice is to
+include a professional prefix. You can choose an icon from the library [here].
+
+The Panel will need some configuration options so the user can choose the collection and fields from that collection to
+include on the panel.
+
+Replace the existing text field with the following fields inside the `options` array:
+
+```js
+{
+	field: 'collection',
+	type: 'string',
+	name: '$t:collection',
+	meta: {
+		interface: 'system-collection',
+		options: {
+			includeSystem: true,
+			includeSingleton: false,
+		},
+		width: 'half',
+},
+},
+{
+	field: 'fields',
+	type: 'string',
+	name: 'Included Fields',
+	meta: {
+		interface: 'system-field',
+		options: {
+			collectionField: 'collection',
+			multiple: true,
+		},
+		width: 'half',
+	},
+},
+{
+	field: 'response_format',
+	name: 'Response',
+	type: 'string',
+	meta: {
+		interface: 'system-display-template',
+		options: {
+			collectionField: 'collection',
+			placeholder: '{{ field }}',
+		},
+		width: 'full',
+	},
+},
+```
+
+After the `options` section, there is the ability to limit the width and height of the panel. Since this panel will hold
+a lot of data, set these to `24` for the width and `18` for the height:
+
+```js
+minWidth: 24,
+minHeight: 18,
+skipUndefinedKeys: ['response_format'],
+```
+
+The output of these options will look like this:
+
+<img src="https://marketing.directus.app/assets/840bb61a-46ff-4474-a026-da5191f1272b.png" alt="Form with collections and fields selection." style="padding: 0 8px 8px 10px">
```

Comment:
Those inline styles are unfortunate. Is that image padding something we can add globally to images, or does this particular screenshot have to be a one-off?

Reply:
Yeah these are ad-hoc when the source image doesn't look amazing. I didn't produce the screenshots or I'd generate them again. This felt preferable and is actually the case across a few of these guides.

Reply:
Maybe a global class for `padding: 0 8px 8px 10px` could bring some consistency? 🤔

Reply:
No - it really is by-eye and not the same every time. Short of going through and regenerating all of these images (sometimes recreating the extensions to do it), this is the best solution.